insights/ui/locators/base.py

```diff
@@ -179,8 +179,88 @@
     "planner.all.button": (
         By.XPATH, ("//div[contains(@class, 'col-sm-4 col-md-7')]")
     ),
-    "planner.yes.popup.button" :(
-        By.XPATH,("//button[contains(.,'Yes')]")
-    )
-})
+    "planner.yes.popup.button": (
+        By.XPATH, ("//button[contains(.,'Yes')]")
+    ),
+
+    # Elements under Configuration page
+    "conf.hidden.rule": (
+        By.XPATH, ("//ul[@class= 'nav nav-faux-tabs']//li[1]")
```

Comment:
Check if we can use some other strategy to find element instead of using index in XPATH.